src/Chapter1.hs

```diff
@@ -473,7 +474,8 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integral a => a -> a
+lastDigit = abs . flip rem 10
```

Comment:
That is an interesting solution! You mastered composition already 👏🏼 

We don't cover this in the course, but functions can be used in the operator style when you put it in the backticks:

```haskell
rem a b
a `rem` b
```

The above two are the same!

So, here, instead of doing `flip` to partially apply the second argument, you can also write
```haskell
flip rem 10
`rem` 10
```